package.json

```diff
@@ -35,15 +36,14 @@
     "karma-chrome-launcher": "2.0.0",
     "karma-sauce-launcher": "^1.1.0",
     "nwb": "^0.15.6",
-    "preact": "^7.2.0",
+    "preact": "*",
     "semantic-release": "^6.3.2",
     "semistandard": "^9.0.0",
-    "skatejs-web-components": "5.2.1",
     "typescript": "^2.1.4",
     "typescript-formatter": "^5.0.0"
   },
   "peerDependencies": {
-    "preact": "^7.2.0"
+    "preact": "*"
```

Comment:
Cool, thanks for that. There's such a minimal API surface that we actually use (we really only just render and directly export h) that I'm not too worried about it at this point. If there's issues we can revisit without affecting our API, most likely.